src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is incorrectly self-closed. In HTML, form elements should not be self-closing. Remove the `/` at the end of the opening tag to ensure proper form functionality.